### ISSUE TITLE
R&D: Investigate how to integrate watermarking and disabling download for VRR

### ISSUE THREAD

**Issue by gamontoya** [ucsdlib/damsmanager#299]
### Descriptive summary

Need to determine the least invasive way to:
- integrate watermarking in the derivative creation process (FFmpeg). Ideally watermark process done before tile creation.
    - Matt P. and Heather S. to come up with watermarking mockups
- disable downloading of files (no download button) for VRR-only collections.

### Visibility/Access Proposal

From Greg R:

- virtual_reading_room_curator
- virtual_reading_room_metadata-only

**Comment by mcritchlow** [comment]
re: disabling download. I think whoever works on this should (hopefully) be able to look at the current user, and see if the context is VRR (do they have a work authorization allowing them to see this record). See the updated DB schema https://github.com/ucsdlib/damspas/blob/feature/vrr/db/schema.rb#L75-L83

**Comment by hweng** [comment]
@gamontoya  Some updates for watermarking:

We tried using ImageMagick to add watermark to vrr image and pdfs. I found that there is ImageMagick commands to do the task to watermark IMAGE. However Dan LaSusa and I found ImageMagick is to be better at handling/converting IMAGES, but not really made to deal with .pdfs in a meaning/smart way. It need to be handled in programming language in python, Java or Ruby.  Dan wrote a python script to add the watermark to PDFs: https://github.com/ucsdlib/watermark
We discussed with Matt Peter, that he'd like the watermarking all happens during ingest in damsmanager. 

So based on @mcritchlow and @jessicahilt suggestions, I've created the following two tickets to follow up watermark solution.

https://github.com/ucsdlib/damsmanager/issues/320
https://github.com/ucsdlib/damspas/issues/623
